zookeeper-recipes/zookeeper-recipes-queue/src/test/java/org/apache/zookeeper/recipes/queue/DistributedQueueTest.java

```diff
@@ -27,243 +28,260 @@
-            queueHandles[i] = new DistributedQueue(clients[i], dir, null);
-        }
-
-        queueHandles[0].offer(testString.getBytes());
-
-        byte[] dequeuedBytes = queueHandles[1].remove();
-        Assert.assertEquals(new String(dequeuedBytes), testString);
+    queueHandles[0].offer(testString.getBytes(UTF_8));
+
+    byte[] dequeuedBytes = queueHandles[1].remove();
+    Assert.assertEquals(new String(dequeuedBytes, UTF_8), testString);
+  }
+
+  @Test
+  public void testTake1() throws Exception {
+    String dir = "/testTake1";
+    String testString = "Hello World";
+    final int numClients = 1;
+    ZooKeeper[] clients = new ZooKeeper[numClients];
+    DistributedQueue[] queueHandles = new DistributedQueue[numClients];
+    for (int i = 0; i < clients.length; i++) {
+      clients[i] = createClient();
+      queueHandles[i] = new DistributedQueue(clients[i], dir, null);
     }
 
-    @Test
-    public void testTake1() throws Exception {
-        String dir = "/testTake1";
-        String testString = "Hello World";
-        final int numClients = 1;
-        ZooKeeper[] clients = new ZooKeeper[numClients];
-        DistributedQueue[] queueHandles = new DistributedQueue[numClients];
-        for (int i = 0; i < clients.length; i++) {
-            clients[i] = createClient();
-            queueHandles[i] = new DistributedQueue(clients[i], dir, null);
-        }
-
-        queueHandles[0].offer(testString.getBytes());
-
-        byte[] dequeuedBytes = queueHandles[0].take();
-        Assert.assertEquals(new String(dequeuedBytes), testString);
+    queueHandles[0].offer(testString.getBytes(UTF_8));
+
+    byte[] dequeuedBytes = queueHandles[0].take();
+    Assert.assertEquals(new String(dequeuedBytes, UTF_8), testString);
+  }
+
+  @Test
+  public void testRemove1() throws Exception {
+    String dir = "/testRemove1";
+    String testString = "Hello World";
+    final int numClients = 1;
+    ZooKeeper[] clients = new ZooKeeper[numClients];
+    DistributedQueue[] queueHandles = new DistributedQueue[numClients];
+    for (int i = 0; i < clients.length; i++) {
+      clients[i] = createClient();
+      queueHandles[i] = new DistributedQueue(clients[i], dir, null);
     }
 
-    @Test
-    public void testRemove1() throws Exception {
-        String dir = "/testRemove1";
-        final int numClients = 1;
-        ZooKeeper[] clients = new ZooKeeper[numClients];
-        DistributedQueue[] queueHandles = new DistributedQueue[numClients];
-        for (int i = 0; i < clients.length; i++) {
-            clients[i] = createClient();
-            queueHandles[i] = new DistributedQueue(clients[i], dir, null);
-        }
-
-        try {
-            queueHandles[0].remove();
-        } catch (NoSuchElementException e) {
-            return;
-        }
-
-        Assert.fail();
+    try {
+      queueHandles[0].remove();
+    } catch (NoSuchElementException e) {
+      return;
+    }
+    Assert.assertTrue(false);
```

Comment:
Not sure why this was changed, but it should be changed back:

```suggestion
    Assert.fail();
```

(or even better, use assertThrows instead, but that's outside the scope of this changeset)

Reply:
@ctubbsii Thanks for the review.  I'm not sure how these changes came to be. I don't think I changed them manually and instead came from something I did merging this relatively old branch into master branch.  I am working on taking into account some of your feedback now and these types of changes were reverted when I tried to rebase on master again.  Thanks.